web/models/package.ex

```diff
@@ -122,8 +122,7 @@ defmodule HexWeb.Package do
   def all_with_docs do
     from(p in Package,
          order_by: p.name,
-         where: not is_nil(p.docs_updated_at),
-         select: {p.name, p.docs_updated_at})
+         where: not is_nil(p.docs_updated_at))
```

Comment:
We should keep the select because we want to keep the data transfer small. We are fetching every package in the database.